samples/doris-demo/flink-demo-v1.1/src/main/java/org/apache/doris/demo/flink/DorisFlinkConnectorDemoV1.java

```diff
@@ -0,0 +1,88 @@
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.apache.doris.demo.flink;
+
+
+import org.apache.doris.flink.cfg.DorisExecutionOptions;
+import org.apache.doris.flink.cfg.DorisOptions;
+import org.apache.doris.flink.cfg.DorisReadOptions;
+import org.apache.doris.flink.sink.DorisSink;
+import org.apache.doris.flink.sink.writer.SimpleStringSerializer;
+import org.apache.flink.api.common.RuntimeExecutionMode;
+import org.apache.flink.api.common.restartstrategy.RestartStrategies;
+import org.apache.flink.api.common.time.Time;
+import org.apache.flink.streaming.api.TimeCharacteristic;
+import org.apache.flink.streaming.api.environment.CheckpointConfig;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
+import java.util.Properties;
+
+/**
+ * Flink doris connector 1.1 demo sample
+ */
+public class DorisFlinkConnectorDemoV1 {
+
+    public static void main(String[] args) throws Exception {
+
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+        env.setStreamTimeCharacteristic(TimeCharacteristic.EventTime);
+        env.setRuntimeMode(RuntimeExecutionMode.BATCH);
+        env.enableCheckpointing(10000);
+        env.getCheckpointConfig().enableExternalizedCheckpoints(CheckpointConfig.ExternalizedCheckpointCleanup.RETAIN_ON_CANCELLATION);
+        env.setRestartStrategy(RestartStrategies.fixedDelayRestart(5, Time.milliseconds(30000)));
+
+
+        DorisSink.Builder<String> builder = DorisSink.builder();
+        final DorisReadOptions.Builder readOptionBuilder = DorisReadOptions.builder();
+        readOptionBuilder.setDeserializeArrowAsync(false)
+            .setDeserializeQueueSize(64)
+            .setExecMemLimit(2147483648L)
+            .setRequestQueryTimeoutS(3600)
+            .setRequestBatchSize(1000)
+            .setRequestConnectTimeoutMs(10000)
+            .setRequestReadTimeoutMs(10000)
+            .setRequestRetries(3)
+            .setRequestTabletSize(1024 * 1024);
+        Properties pro = new Properties();
+        pro.setProperty("format", "json");
+        pro.setProperty("read_json_by_line", "true");
+        pro.setProperty("line_delimiter", "\n");
+        DorisOptions.Builder dorisBuilder = DorisOptions.builder();
+        dorisBuilder.setFenodes("FE_IP:8030")
+            .setTableIdentifier("test.test_flink")
+            .setUsername("root")
+            .setPassword("");
+        DorisExecutionOptions.Builder  executionBuilder = DorisExecutionOptions.builder();
+        executionBuilder
+            .setStreamLoadProp(pro)
+            .setLabelPrefix("doris_test")
+            .setBufferSize(8*1024)
+            .setBufferCount(3);
```

Comment:
These two parameters are not recommended for users to set, it is recommended to delete